baton.lua

```diff
@@ -139,12 +169,25 @@ function Player:update()
 end
 
 function Player:getRaw(name)
-  return self._controls[name].value
+  return unpack(self._controls[name].values)
 end
 
 function Player:get(name)
-  local v = self:getRaw(name)
-  return v > self.deadzone and v or 0
+  local a, b = self:getRaw(name)
+  if not self:_isMagnitudeGreaterThanDeadzone(a, b) then
+    a = 0
+    b = b and 0 or nil
+  end
```

Comment:
`return a` and `return a, nil` are actually a little bit different in obscure circumstances.